gpfit/print_fit.py

```diff
@@ -10,10 +10,10 @@ def print_ISMA(A, B, alpha, d, K):
             print printString
             printString = '    + '
 
-        printString += '({0:.3g}/w**{1:.3g})'.format(exp(alpha[k] * B[k]), alpha[k])
```

Comment:
It may have been a bad choice, but it was chosen so that the log-space variables (the focus of that paper) were x and y.

I don't advocate writing all GP papers in terms of u and w, but I think for the fitting work (and by extension, GPfit), we should stay with the notation used in our paper.